usaspending_api/references/management/commands/update_location_usage_flags.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Count, Case, F, Value, When, Q
+from usaspending_api.references.models import Location
+import logging
+import django
+
+
+class Command(BaseCommand):
+    help = "Updates Location object usage flags based upon their current usage."
+    logger = logging.getLogger('console')
+
+    def handle(self, *args, **options):
+        annotated_set = Location.objects.annotate(legalentity_count=Count('legalentity'))
+        annotated_set = annotated_set.annotate(award_count=Count('award'))
```

Comment:
Yeah, if we move toward the more simplified version of the update, it would be no biggie to handle the award table too.